frontend/src/pages/AddPage.tsx

```diff
@@ -0,0 +1,362 @@
+        addInsurance();
+    };
+
+    return (
+        <>
+            <h2>Neue Versicherung hinzufügen </h2>
+            <div className="add-insurance">
+                <form onSubmit={handleSubmit} className="insurance-form">
+                    <div className="form-section">
+                        <label>
+                            Vorname: <input
+                            type="text"
+                            name="firstName"
+                            value={firstName}
+                            onChange={(event) => setFirstName(event.target.value)}
+                            required
+                        />
+                        </label>
+                        <br/>
+
+                        <label>
+                            Name: <input
+                            type="text"
+                            name="familyName"
+                            value={familyName}
+                            onChange={(event) => setFamilyName(event.target.value)}
+                        />
+                        </label>
+                        <br/>
+
+                        <label>
+                            PLZ: <input
+                            type="text"
+                            name="zipCode"
+                            value={zipCode}
+                            onChange={(event) => setZipCode(event.target.value)}
+                        />
+                        </label>
+                        <br/>
+
+                        <label>
+                            Ort: <input
+                            type="text"
+                            name="city"
+                            value={city}
+                            onChange={(event) => setCity(event.target.value)}
+                        />
+                        </label>
+                        <br/>
+
+                        <label>
+                            Adresse: <input
+                            type="text"
+                            name="address"
+                            value={address}
+                            onChange={(event) => setAddress(event.target.value)}
+                        />
+                        </label>
+                        <br/>
+
+                        <label>
+                            Telefon: <input
+                            type="text"
+                            name="telephone"
+                            value={telephone}
+                            onChange={(event) => setTelephone(event.target.value)}
+                        />
+                        </label>
+                        <br/>
+
+                        <label>
+                            E-Mail: <input
+                            type="text"
+                            name="email"
+                            value={email}
+                            onChange={(event) => setEmail(event.target.value)}
+                        />
+                        </label>
```

Comment:
This is a lot of the same code. A component reduces this part a lot. You could do something like that:

```tsx
<FormLabel name="telephone" value={telephone} handleOnChange={setTelephone} />
<br/>
<FormLabel name="email" value={email} handleOnChange={setEmail} />
```